lib/galaxy/job_metrics/instrumenters/hostname.py

```diff
@@ -1,23 +1,17 @@
 """The module describes the ``hostname`` job metrics plugin."""
 
 import logging
+from typing import Any
```

Comment:
```suggestion
```

Reply:
@nsoranzo You are more than welcome to just push to branches in the future -- it's way easier ;)